src/main/java/com/clanki/objects/FlashcardList.java

```diff
@@ -3,7 +3,7 @@
 import java.util.ArrayList;
 
 public class FlashcardList {
-    private ArrayList<Flashcard> flashcards;
+    private final ArrayList<Flashcard> flashcards;
```

Comment:
Why is `flashcards` a `final` variable?

Reply:
@SongZijin Hi, is this supposed to be final?